lua/entities/gmod_wire_expression2/core/custom/pewpewfunctions.lua

```diff
@@ -455,4 +455,17 @@ e2function entity pewCreate( string bullet, string model, vector pos, angle ang
 	return create(self.player, bullet, model, pos, ang)
 end
 
+
+__e2setcost(0)
+
+e2function number timeSinceLastFired( entity cannon )
+    if not IsValid(cannon) then return 0 end
```

Comment:
```suggestion
    if not IsValid(cannon) then return -1 end
```

While I don't think it's a big deal because `0` is just as invalid as `-1`, I think it's clearer that something went wrong if it's `-1`, whereas `0` could seem like a bug of some kind